README.md

```diff
@@ -27,26 +27,45 @@ This action enables you to get the PR no matter which event type triggered the w
 
 ## :keyboard: Usage
 
-```
+```yml
   steps:
-    - uses: actions/checkout@v1
+    - uses: 8BitJonny/gh-get-current-pr@2.0.0
+      id: PR
+
+    - run: echo "Your PR number is ${{ steps.PR.outputs.number }} and its JSON is ${{ steps.PR.outputs.pr }}"
+```
+
+### Inputs
+See [action.yml](action.yml) for more details.
+```yml
```

Comment:
Can we keep it uniform? The first code snippet starts with 'steps', this one skips it, the "Outputs" one has it again and the "Workaround" code snipped is skipping the 'steps' line again.

I don't have too much of an opinion on whether it cleaner with or without the 'steps' line but I kinda like it to be uniform. Can you change that for all the code snippets?

Reply:
oh, yes, also wanted to fix the indentation so when copied to yaml, it's at the correct level. I'll do the changes today or tomorrow.

Reply:
added `id: PR` to all as well